performance-test/support-files/template_type.txt

```diff
@@ -0,0 +1,2 @@
+${TemplateType}
+mosip.email.template.property
```

Comment:
I see the below pdf file is password protected, what is this pdf file actually ?

Reply:
Can we use a file which is not password protected, so we don't get issues with virus scanning ?

Reply:
Replaced with the file which is not password protected.

Reply:
Reviewed. Passoword protection has been removed.